docs/src/pages/[platform]/connected-components/account-settings/change-password/react.mdx

```diff
@@ -1,3 +1,359 @@
-## Change Password
+import { Alert, Expander, ExpanderItem } from '@aws-amplify/ui-react';
 
-TODO
+import ReactPropsTable from '@/components/propsTable/ReactPropsTable';
+import { Example, ExampleCode } from '@/components/Example';
+import { ERROR_MESSAGE } from '../props';
+import { CHANGE_PASSWORD, OVERRIDES, PASSWORD_FIELDS, SUBMIT_BUTTON } from './props';
+
+
+### Props
+
+<ReactPropsTable props={CHANGE_PASSWORD} />
+
+### Basic Usage
+
+`ChangePassword` has `onSuccess` handler that will be called after successful password change. You may use this callback to run any custom handling, such as navigating route or rendering confirmation messages.
+
+<Example>
+  <ExampleCode>
+
+```jsx
+import React from 'react';
+import { AccountSettings } from '@aws-amplify/ui-react';
+
+function App() {
+  const handleSuccess = () => {
+    alert('password is successfully changed!')
+  }
+  
+  return (
+    <AccountSettings.ChangePassword onSuccess={handleSuccess}/>
+  );
+}
+```
+
+  </ExampleCode>
+</Example>
+
+### Custom Validation
+
+You can override the default custom password validator with your own. To do so, you can use the validators prop, which takes an array of ValidatorOptions, which is of type:
+
+<Example>
+  <ExampleCode>
+
+```ts
+interface ValidatorOptions {
+  validationMode: 'onBlur' | 'onChange' | 'onTouched';
+  validator: (value: string) => boolean;
+  message: string;
+}
+```
+
+  </ExampleCode>
+</Example>
+
+- `onBlur` validates password on every blur event
+- `onChange` validates password on every change event
+- `onTouched` validates password on first blur, and on every change event after the first blur.
+
+For example, following is a minLength validator that validates on every change:
+
+<Example>
+  <ExampleCode>
+
+```js
+const minLength = {
+  validationMode: 'onChange',
+  validator: (password) => password.length >= 4,
+  message: 'Password must have length 4 or greater',
+};
+```
+
+  </ExampleCode>
+</Example>
+
+You can pass multiple validators to ConfirmPassword to override the default validator:
+
+<Example>
+  <ExampleCode>
+
+```jsx
+import React from 'react';
+import { AccountSettings } from '@aws-amplify/ui-react';
+
+function App() {
+  const minLength = {
+    validationMode: 'onChange',
+    validator: (password) => password.length >= 4,
+    message: 'Password must have length 4 or greater',
+  };
+  
+  const handleSuccess = () => {
+    alert('password is successfully changed!')
+  }
+  
+  return (
+    <AccountSettings.ChangePassword onSuccess={handleSuccess} validators={[minLength]}/>
```

Comment:
Can we actually show multiple validators being passed here, in addition to `minLength`?

Reply:
Done in 65388d0a1!